Sources_v3/StreamChatUI/Utils/UIImage+Extensions.swift

```diff
@@ -9,3 +9,19 @@ extension UIImage {
         self.init(named: name, in: bundle, compatibleWith: nil)
     }
 }
+
+extension UIImage {
+    func tinted(with fillColor: UIColor) -> UIImage? {
+        let image = withRenderingMode(.alwaysTemplate)
+        UIGraphicsBeginImageContextWithOptions(size, false, scale)
+        fillColor.set()
+        image.draw(in: CGRect(origin: .zero, size: size))
+
+        guard let imageColored = UIGraphicsGetImageFromCurrentImageContext() else {
+            return nil
+        }
+        
+        UIGraphicsEndImageContext()
+        return imageColored
+    }
+}
```

Comment:
This a nasty hack 🙈 We don't need this at all. `UIImageView` will render template images correctly based on its tint color.

Reply:
After discussion we decided that this could be a valid approach.